src/libraries/Microsoft.Win32.Registry/src/Microsoft/Win32/RegistryKey.Windows.cs

```diff
@@ -197,7 +197,7 @@ private void DeleteValueCore(string name, bool throwOnMissingValue)
         /// <returns>The RegistryKey requested.</returns>
         private static RegistryKey OpenBaseKeyCore(RegistryHive hKeyHive, RegistryView view)
         {
-            IntPtr hKey = (IntPtr)((int)hKeyHive);
+            nint hKey = (int)hKeyHive;
```

Comment:
`hKey` is a handle, so `void*` might be better.

Reply:
CS0030: Cannot convert type 'RegistryHive' to 'void*'